### PR TITLE
docs: some improvements

### PR DESCRIPTION
**Pull request by deadprogram**
This PR adds some small documentation improvements to hopefully make it a little easier for people to get started.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,33 @@ A minimal terminal for TinyGo devices supporting 256-color ANSI escape codes.
...
+```
+
+### basic-clue
+
+Displays basic text. Runs on the Clue using software scrolling.
+
+```
+tinygo flash -target clue ./examples/basic-clue
+```
+
```
Comment (sago35) [inline README.md]: #7 may require some minor documentation changes.

Comment (deadprogram) [reply README.md]: Understood. Let's get that one done first, and then I will modify this PR accordingly.

Comment (deadprogram) [reply README.md]: Updated.

**Comment by deadprogram** [comment]
I have updated this PR based on the latest changes @sago35 please take a look.

**Review by sago35 — APPROVED** [review]
LGTM

**Comment by deadprogram** [comment]
Thanks for review @sago35 now merging.

**Pull request merged**